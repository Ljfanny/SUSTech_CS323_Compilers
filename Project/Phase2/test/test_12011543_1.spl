int compare(int x, int y)
{
  return 1;
  return 2;
  return 3;
  return 4;
  return 5;
  return 6;
}
float compare(int x, int y)
{
  return 1;
  return 2;
  return 3;
  return 4;
  return 5;
  return 6;
}
float compare2(int a, int b){
  return 1.0;
  return 2.0;
  return 3.0;
  return 4.0;
  return 5.0;
  return 6.11;
}
int test_12011543_1()
{
  int a, b, c, k;
  int f = b + s;
  int d =!!!(a = !!(b = !(c = 0))) ;
  int r = add(k);
  return compare(7, 8);
}