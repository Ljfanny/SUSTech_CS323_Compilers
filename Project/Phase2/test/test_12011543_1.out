Error type 4 at Line 10: redefine function: compare
Error type 1 at Line 30: undefined variable: s
Error type 7 at Line 30: unmatching operands: PLUS
Error type 5 at Line 30: unmatching type on both sides of assignment
Error type 2 at Line 32: undefined function: add
Error type 5 at Line 32: unmatching type on both sides of assignment
